.github/workflows/CompatHelper.yml

```diff
@@ -13,4 +12,5 @@ jobs:
       - name: CompatHelper.main()
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COMPATHELPER_PRIV: ${{ secrets.COMPATHELPER_PRIV }}
```

Comment:
This requires a new `COMPATHELPER_PRIV` secret (see https://github.com/JuliaRegistries/CompatHelper.jl#12-set-up-the-ssh-deploy-key-optional).

Reply:
Can you fix this @cpfiffer? Or have you already added one?

Reply:
Nope, crap, I missed that. I'll add one now.

Reply:
Added.

Reply:
Thank you!